discord/iterators.py

```diff
@@ -163,7 +163,10 @@ async def fill_users(self):
             else:
                 for element in reversed(data):
                     member_id = int(element['id'])
-                    member = self.guild.get_member(member_id)
+                    try:
+                        member = self.guild.get_member(member_id)
+                    except AttributeError:
+                        member = None
```

Comment:
```suggestion
                    member = self.guild.get_member(member_id)
```

Then in line 160 instead of `if self.guild is None` change it to `if self.guild is None or isinstance(self.guild, Object):`